themes/bootstrap3/js/common.js

```diff
@@ -239,21 +239,52 @@ var VuFind = (function VuFind() {
     return html.replace(/(<script[^>]*) nonce=["'].*?["']/ig, '$1 nonce="' + getCspNonce() + '"');
   };
 
+  function setInnerHtml(elm, html) {
+    elm.innerHTML = html;
+    Array.from(elm.querySelectorAll("script")).forEach(oldScript => {
```

Comment:
It would probably be a good idea to add a comment here to explain the purpose of this code. For example, something like:

```
// If the loaded HTML includes script tags, extra work is needed to ensure that they execute:
```